barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_flavor.hpp

```diff
@@ -183,15 +179,15 @@ class UltraFlavor {
                               lookup_read_tags)   // column 7
 
         auto get_wires() { return RefArray{ w_l, w_r, w_o, w_4 }; };
-        auto get_to_be_shifted() { return RefArray{ z_perm }; };
+        auto get_to_be_shifted() { return RefArray{ w_l, w_r, w_o, w_4, z_perm }; };
```

Comment:
again, weird. Is this somehow equivalent to what we had before? was this jsut not used?